### PR TITLE
Clean up translations.

### PR DESCRIPTION
**Pull request by kialam**
The original cause of the bug was due to an `i18nKey` prop being provided to a block of text that was wrapped in the `<Trans>` component without its corresponding entry in translations.json. Upon further reading of the docs, it is stated that the `<Trans>` component is [rarely needed](https://react.i18next.com/latest/trans-component#important-note), and that most strings can be marked for translation using the provided `t` macro.

This PR cleans up some translations and help text blocks. Previously, the `<Trans>` component was used to wrap long help text blocks with formatting. I have used the `t` macro in areas where possible, and in other areas where strings include HTML text formatting, I have wrapped using the provided `<Trans>` component.

### CODE REVIEW

**Review by rooftopcellist — APPROVED** [review]

File: frontend/awx/dashboard/AwxDashboard.tsx [hunk 1]
```diff
@@ -40,12 +40,13 @@ export function AwxDashboard() {
...
-          <Trans>
-            <p>
-              <InfoCircleIcon /> You are currently viewing a tech preview of the new {product} user
-              interface. To return to the original interface, click <a href="/">here</a>.
-            </p>
-          </Trans>
+          <p>
+            <InfoCircleIcon />{' '}
+            <Trans>
+              You are currently viewing a tech preview of the new {{ product }} user interface. To
```
Comment (vidyanambiar) [inline frontend/awx/dashboard/AwxDashboard.tsx:46]: Do we need a translation key here for the content within this Trans component to show up in the translation file?

Comment (kialam) [reply frontend/awx/dashboard/AwxDashboard.tsx:46]: We do not; the `i18nKey` prop is optional. I ran `npm run i18n` and it produced the following strings in the `translation.json` file:

```
"You are currently viewing a tech preview of the new {{product}} user interface. To return to the original interface, click <3>here</3>.": "You are currently viewing a tech preview of the new {{product}} user interface. To return to the original interface, click <3>here</3>.",
```

Comment (vidyanambiar) [reply frontend/awx/dashboard/AwxDashboard.tsx:46]: Great! Thanks for checking 👍

**Review by vidyanambiar — APPROVED** [review]
lgtm

**Pull request merged**